frontend/src/components/molecules/EmptyState/index.jsx

```diff
@@ -0,0 +1,30 @@
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateHeader,
+  EmptyStateIcon,
+} from "@patternfly/react-core";
+
+import PropTypes from "prop-types";
+import SearchIcon from "@patternfly/react-icons/dist/esm/icons/search-icon";
+
+const msgToShow = {
+  noData: "No data found for the selected date range.",
+  noFilterData: "No results match the filter criteria.",
+};
+const CustomEmptyState = ({ type }) => (
+  <EmptyState data-testid="custom_empty_state">
```

Comment:
I believe this should be `ouiaId`, but probably `data-ouia-component-id` since it doesn't inherit from a patternfly component. The corresponding test should also reference that instead of `data-testid`.